src/logos/satispay/logo.jsx

```diff
@@ -0,0 +1,65 @@
+/* @flow */
+/** @jsx node */
+
+import { node, type ComponentNode } from "@krakenjs/jsx-pragmatic/src";
+
+import { SVGLogo, getLogoColors, type SVGLogoProps } from "../../lib";
+import { LOGO_COLOR, LOGO } from "../../constants";
+import { type LogoColorMap } from "../../types";
+
+const LOGO_COLORS: LogoColorMap = {
+  [LOGO_COLOR.DEFAULT]: {
+    primary: "#2C2E2F",
```

Comment:
Based on the UX design, the default logo could should be white rendered on the red background. The way you have it configured shows a dark color logo on the default red button background 
```suggestion
    primary: "#FFFFFF",
```
<img width="937" alt="Screen Shot 2022-11-01 at 5 35 25 PM" src="https://user-images.githubusercontent.com/9424291/199367484-22da6fb0-850e-4e90-a989-86b686605513.png">